deploy/manifest/built-in-providers/applications_core.yaml

```diff
@@ -4,34 +4,34 @@ types:
     apiVersions:
       "2025-01-01-preview":
         schema: {}
-        capabilities: []
+    capabilities: []
```

Comment:
Note: we decided to move this to the resource type rather than the API version. 

There are edge cases in how a consumer should behave for a capability that was **added** to a resource type, and is missing support from previous API versions. These edge cases are present regardless of where we configure the capability, so we went with the simplest design.

Reply:
A lot of the changes in this PR are updates to existing configuration files or tests.